### ISSUE TITLE
ktlint as the formatter in detekt

### ISSUE THREAD

**Issue by kaushalyap** [arturbosch/detekt#1941]
[Question](https://stackoverflow.com/questions/57336354/ktlint-as-the-formatter-in-detekt) was asked on SO, but it seem that there is a possibility of a bug

Since generated html report shows no findings when following line is uncommented.

`config = files(file("$project.rootDir/tools/detekt.yml"))`

**Comment by arturbosch** [comment]
Hi, sorry for the laste response.

Do you have the formatting config block in your configuration file?
```yaml
formatting:
  active: true
  android: false
  autoCorrect: true
``

If you delete the `config = ...` line, the default config is used. This may be reason for it.
Consider using `--build-upon-default-config` to merge your settings and the default ones.
`buildUponDefaultConfig = true` in gradle.

**Comment by kaushalyap** [comment]
Adding following fixed the issue (findings are visible, with config uncommented)
> buildUponDefaultConfig = true

**Comment by kaushalyap** [comment]
@arturbosch Should I set following in order to [set ktlint as the formatter with detekt](https://github.com/arturbosch/detekt#adding-more-rule-sets)
> formatting:
>   active: true
>   android: false
>   autoCorrect: true

**Comment by arturbosch** [comment]
Not needed if you have `buildUponDefaultConfig`. These should be preset for you.

**Comment by kaushalyap** [comment]
> Not needed if you have `buildUponDefaultConfig`. These should be preset for you.

so I just have to add following to make it work?

`detektPlugins "io.gitlab.arturbosch.detekt:detekt-formatting:[version]"`
